tck/src/main/java/org/eclipse/microprofile/rest/client/tck/InvokeWithBuiltProvidersTest.java

```diff
@@ -94,7 +94,7 @@ public void testInvokesPutOperationWithRegisteredProviders() throws Exception {
 
         InterfaceWithoutProvidersDefined api = createClient();
 
-        Response response = api.executePut(id, inputBody);
+        Response response = api.executePut(expectedId, inputBody);
```

Comment:
This test checks that the providers are invoked, not the server behavior.

Reply:
The wiremock setup in this test registers `/expectedId`, hence any invocation to `/id` fails with a `404` .

Reply:
Yeah, I figured that's what the issue was.  I'm not against making the change, but your impl should pass without this.

Reply:
Actually, now that I'm thinking about it, the CXF impl should be seeing the 404 getting thrown as an exception as well (assuming that's what you're seeing in your impl)

Reply:
I see, thanks for the clarification. It seems I missed that subtle involvement of the param  converter